test/index.spec.ts

```diff
@@ -1,4 +1,5 @@
 import path from 'path';
+import { type } from 'os';
```

Comment:
Used or not; if importing consider using an alias to prevent keyword overload confusion